src/html/index.html

```diff
@@ -0,0 +1,71 @@
+{#
+  Nunjucks Templating!
+  Homepage: https://mozilla.github.io/nunjucks/
+  Docs: https://mozilla.github.io/nunjucks/api.html
+#}
+
+{% extends 'layouts/application.html' %}
+
+{% block content %}
+<div id="top-fold">
+  <div id="title-box">
+  <p id="subtitle">A Viget Exploration</p>
+    <h1 id="title">
+      <a href="/">
+        <img src="images/title.svg" alt="Improving Medical Adherence" />
+      </a>
+    </h1>
+    <img src="images/yellow-pill.svg" alt="Yellow pill." id="yellow-pill">
```

Comment:
Probably don't need to have alt attributes on these pill images, since they're just decorative in nature. Screen readers will read alt attributes, which is good in most cases, but for decorative images it's recommended to just have an empty alt attribute, e.g. `alt=""` so that screen readers will just skip over the image.